utilities/src/main/java/io/syndesis/qe/cucumber/MailFormatter.java

```diff
@@ -108,6 +108,22 @@ private void onTestRunFinished(TestRunFinished t) {
             for (String line : results.stream().map(ScenarioResult::toString).collect(Collectors.toList())) {
                 out.write(line + " <br/>\n");
             }
+            out.write("<h2>All open issues: </h2>");
+            out.write("<ul>");
+            List<SimpleIssue> allOpenIssues = new ArrayList<>();
+            results.forEach(scenarioResult -> {
+                if (scenarioResult.getIssues() != null) {
+                    scenarioResult.getIssues().forEach(issue -> {
```

Comment:
you can use `allOpenIssues.addAll(...stream().filter(...).collect(to list)), but that is pretty much the same :dancer:

Reply:
I did flatStream on results before but that caused issues on Jenkins so I went oldschool :space_invader: